---
layout: archive
title: "Projects"
permalink: /projects/
author_profile: true
---

### Projects at UT Southwestern Medical Center, 2016/01 - present
1. **A New Method to Identify Somatic Copy Number Alternations in Tumors**  
    SCNAs were observed frequently in many kinds of tumors. SCNA detection using high-throughput sequencing was becoming popular in recent years. Several methods have been developed for SCNA detection using exome-sequencing data. However, there is a common problem for most of these methods. To give an estimation of copy number, a strong assumption was usually adopted: most regions of the genome don’t have SCNAs. But that assumption is not necessary true in tumor cells, because large scale copy number alterations were observed frequently in tumors. Therefore, I developed a new method to detect SCNAs in tumors based on large-scale sequencing data. In this method, both read depth and allele frequency were considered, and it doesn’t rely on the assumption that there are no large-scale SCNAs in tumor cells. In the evaluation using a published dataset, I compared the performance of DEFOR with the other six methods, and DEFOR has better accuracy than the other methods especially in the situation where there are large-scale SCNAs in the unstable tumor genomes.

2. **Genome-wide Mutation Landscape in _POLE<sup>P286R</sup>_ Engineered Mice**  
    Eukaryotic DNA replication is astoundingly accurate, with an error rate of only one per 10<sup>9</sup> to 10<sup>10</sup> bases. DNA polymerase ε (POLE) plays a key role in leading strand DNA synthesis. Inactivation of the exonuclease domain of POLE can cause high error rate during DNA replication. The high mutation rates (greater than 100/Mb) in some cancers are due to somatically-acquired POLE missense mutations in its proofreading domain. _POLE<sup>P286R</sup>_ is the most common recurrent substitution observed in endometrial cancer, and results in the strongest mutator phenotype. _POLE<sup>P286R</sup>_ is in the exonuclease domain, and cause the loss of the proofreading activity of POLE. To study the mutation rate and pattern caused by _POLE<sup>P286R</sup>_, a conditional _POLE<sup>P286R</sup>_ allele was engineered in mice in the lab of my collaborator, Dr Diego Castrillon. I designed a whole genome sequencing study for MEF cell lines in different passages (P15, P30 and P45) and samples from lung adenocarcinomas and T cell lymphomas. I contributed key ideas about sample selection, and finalized all of the data analysis. We found that the average mutation rate in _POLE<sup>P286R</sup>_/+ MEFs was around 1.6 substitutions per megabase per cell division, at least three orders of magnitude higher than the normal DNA replication error rate. _POLE<sup>P286R</sup>_/+ cancers and MEFs showed a high incidence of C>A and C>T substitutions and rare C>G and T>A substitutions. In this project, we estimated the error rate during DNA replication in the mouse with _POLE<sup>P286R</sup>_ mutation, and get a picture of mutation signatures. Through this collaboration, we have better understanding about the role of POLEP286R/+ mutation in cancer development.
    
3. **Ribosome profiling of eIF5A cell lines in human and yeast**  


3. **Genetics of Kidney Cancer**  
    Kidney cancer is among ten most common cancers in United States. To better understand the genetic mechanism of the different types of kidney cancer, Kidney Cancer Project (KCP) was initiated by Dr. James Brugarolas. I am responsible for processing and analyzing all exome-sequencing data and RNA-sequencing data of 697 samples from 219 patients with kidney cancer.
To study the possible genetic mechanism under invasion of tumor cells from primary tumor site, samples from thrombus and primary tumor site were compared to identify the difference in somatic mutations and gene expression levels. Although there is no significant difference between thrombus and primary tumor in terms of somatic mutations, we observed several candidate genes with significantly lower expression level in thrombus, including known tumor suppressor genes (e.g. PCDH10 and EYA4). That indicates that down-regulation of tumor suppressor genes may play an important role during the invasion of tumor cells.

### Projects at University of Michigan, 2012/04 – 2016/01
1. **HUNT Whole Genome Sequencing Project**  
    The purpose of this on-going project is to perform whole genome sequencing in 2,202 samples (1,100 MI cases and 1,102 controls) from Norway, and provide a reference panel for association study, population genetics study and imputation. I am leading the analysis of this project, and have performed read-mapping, genotype-calling, quality control and statistical analysis of the sequencing data.

2. **GeneZoom**  
    GeneZoom is a convenient tool to visualize the association test results at both variant and gene levels. GeneZoom can be used as a program or in our website interface. I participated in the design, development and testing of GeneZoom. The manuscript is in preparation.

3. **PUUMA myocardial infarction (MI) and plasma lipid association study**  
    The goal of this project is to identify variants associated with MI and plasma lipid levels in a Chinese population. 10,030 samples from China were genotyped using Illumina Human Exome-chip. I performed genotype-calling, quality control and computational analysis for these data. We identified four novel Asian-specific variants associated with plasma lipid values.

4. **NHLBI Exome sequencing project (ESP)**  
    The goal of the NHLBI GO Exome Sequencing Project (ESP) is to discover novel genes and mechanisms contributing to heart, lung and blood disorders by pioneering the application of next-generation sequencing of the protein coding regions of the human genome. I performed gene-based burden tests for LDL-cholesterol based on whole exome sequencing data of 5,500 samples. We identified several low-frequency variants with large effect on LDL-cholesterol. A co-first author paper has been published in The American Journal of Human Genetics.

5. **HUNT MI and plasma lipid association study**  
    The goal of this project is to identify variants associated with MI and plasma lipid levels in Norwegians. 6,000 samples from Norway were genotyped using Illumina Human Exome-chip. I performed genotype-calling, quality control and computational analysis for these data. We identified a new causal coding variant in TM6SF2 associated with total cholesterol, and our collaborators verified its function. Two co-first author papers have been published in Nature Genetics and Human Molecular Genetics.

6. GLGC meta-analysis of plasma lipid association study  
    This project involved more than 300,000 samples from 92 studies. I participated in the analysis of ancestry inference, association variants summary and visualization.

### PhD candidate in Peking University, 2006/09 – 2011/07

1. **Phylogeny guided genome alignment method (PGGA)**  
    I developed a method to construct genome alignment and identify duplication and rearrangement in genomes. This method uses a dynamic programming algorithm, and is implemented using C and Perl.

2. **Plant transcription factors database (PlantTFDB)**  
    As one of the major contributors, I participated in the development of a method to identify transcription factors from plant genomes, and constructed PlantTFDB 1.0 and 2.0. PlantTFDB is the most famous database in this area, and it has been cited more than 300 times in total. One first-author paper and two co-author papers have been published on Nucleic Acids Research.

3. **Rice-Map**  
    Rice-Map is a web-based browser used to visualize the genome sequence, gene structure, expression, homology and many other features of the rice genome. Users can browse the genome as browsing a map. I contributed my ideas in the design of the Rice-Map and participated in a part of development.

4. **Evolution and function analysis of DNMT3a and DNMT3b**  
    I performed evolution analysis for DNMT3a and DNMT3b, and identified a positive selected mutation. Our collaborators verified that this mutation plays a key role in the functional divergence of DNMT3a and DNMT3b.

5. **Expression pattern divergence of duplicated genes in rice**  
    I participated in the analysis of gene expression array data from 45 different tissues and conditions in rice development. We found the pattern of expression divergence were different for block duplication, tandem duplication and disperse duplication.
